iOS Example Frame/iOS Example FrameUITests/iOS_Example_FrameUITests.swift

```diff
@@ -2,52 +2,10 @@
 //  iOS_Example_FrameUITests.swift
 //  iOS Example FrameUITests
 //
-//  Created by Floriel Fedry on 11/06/2018.
-//  Copyright © 2018 Checkout. All rights reserved.
+//  Created by Ehab Alsharkawy.
+//  Copyright © 2022 Checkout. All rights reserved.
 //
 
 import XCTest
 
-class iOS_Example_FrameUITests: XCTestCase {
-
-    override func setUp() {
-        super.setUp()
-
-        // Put setup code here. This method is called before the invocation of each test method in the class.
-
-        // In UI tests it is usually best to stop immediately when a failure occurs.
-        continueAfterFailure = false
-        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
-        XCUIApplication().launch()
-
-        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
-    }
-
-    override func tearDown() {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
-        super.tearDown()
-    }
-
-    func testExample() {
-        // Use recording to get started writing UI tests.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
-
-        let app = XCUIApplication()
-        app.buttons["Old"].tap()
-        let elementsQuery = app.scrollViews.otherElements
-        elementsQuery.textFields["4242"].tap()
-        elementsQuery.textFields["4242"].typeText("424242424242424242")
-        elementsQuery.textFields["06/20"].tap()
-        // Potential Task to fix the expiry date in past issue
-        app.pickerWheels.element(boundBy: 1).swipeUp()
-
-        elementsQuery.textFields["100"].tap()
-        elementsQuery.textFields["100"].typeText("100")
-        elementsQuery.staticTexts["Billing Details*"].tap()
-        app.navigationBars["Billing"].buttons["Done"].tap()
-        app.toolbars["Toolbar"].buttons["Done"].tap()
-        app.navigationBars["Payment"].buttons["Pay"].tap()
-        app.alerts["Payment"].buttons["OK"].tap()
-    }
-
-}
+class iOS_Example_FrameUITests: XCTestCase { }
```

Comment:
Could you add a subtask to the current ticket to follow up with this please? Doesn't look very good to leave code like this behind, but I also think the PR is in a good state to get merged so wouldn't want to do it now.

Also include the Carthage UITest in the ticket too 🙏

Reply:
[Jira ticket](https://checkout.atlassian.net/browse/PIMOB-1360?atlOrigin=eyJpIjoiZTQ0ZDJkMjllZDY1NDE2NTkxMzBmYzhkOGZhNTdkZGYiLCJwIjoiaiJ9)